librz/il/theory_fbasic.c

```diff
@@ -286,6 +286,38 @@ void *rz_il_handler_fsqrt(RzILVM *vm, RzILOpPure *op, RzILTypePure *type) {
 	return ret;
 }
 
+void *rz_il_handler_fexcept(RzILVM *vm, RzILOpPure *op, RzILTypePure *type) {
+	rz_return_val_if_fail(vm && op && type, NULL);
+	RzILOpArgsFexcept args = op->op.fexcept;
+	RzFloat *n = rz_il_evaluate_float(vm, args.x);
+	if (!n) {
+		return NULL;
+	}
+
+	bool e = false;
+	switch (args.e) {
+	case RZ_FLOAT_E_DIV_ZERO:
+		e = n->exception & RZ_FLOAT_E_DIV_ZERO;
+		break;
+	case RZ_FLOAT_E_OVERFLOW:
+		e = n->exception & RZ_FLOAT_E_OVERFLOW;
+		break;
+	case RZ_FLOAT_E_UNDERFLOW:
+		e = n->exception & RZ_FLOAT_E_UNDERFLOW;
+		break;
+	case RZ_FLOAT_E_INEXACT:
+		e = n->exception & RZ_FLOAT_E_INEXACT;
+		break;
```

Comment:
This should also generate an exception event like here: https://github.com/rizinorg/rizin/blob/e7b919f001c5894ee66aeae49567e6cf8dfc429a/librz/il/theory_bitv.c#L233